src/flownet/config_parser/_config_parser.py

```diff
@@ -459,6 +463,112 @@ def _to_abs_path(path: Optional[str]) -> str:
 
     config = suite.snapshot
 
+    req_relp_parameters: List[str] = []
+
+    for phase in config.flownet.phases:
+        if phase.lower() not in ["oil", "gas", "water", "disgas", "vapoil"]:
```

Comment:
There are a lof of `.lower()` around. Maybe we could just transform it to lower on initial parse with `ConfigSuite`? (Similarly to how e.g. relative paths are converted to absolute paths). That would the `if` statements.